test/e2e/data/e2e_eks_conf.yaml

```diff
@@ -131,7 +131,7 @@ variables:
   EXP_EKS_IAM: "false"
   EXP_EKS_ADD_ROLES: "false"
   VPC_ADDON_VERSION: "v1.18.3-eksbuild.2"
```

Comment:
@tamalsaha would you be able to change this to `v1.19.5-eksbuild.3` please? That might fix the new error we are encountering.